content/partials/quickstart/publishing-app-store.md

```diff
@@ -38,6 +43,11 @@ publishing:
     # Optional boolean, defaults to false. Whether or not to submit the uploaded
     # build to App Store review. Note: This action is performed during post-processing.
     submit_to_app_store: true
+
+    # Optional boolean, defaults to false. Whether to cancel previous submissions (if applicable)
+    # to App Store Connect. It allows automatically submitting a new submission for review if a previous submission exists.
```

Comment:
Here too I would propose to change as follows. 
"Set to true to cancel the previous submission when submitting a new build to App Store review."
Would it be relevant here to mention which statuses allow canceling the submission? If not here, then in CLI tools docs.